Sources/CasePaths/EnumReflection.swift

```diff
@@ -352,6 +352,10 @@ extension EnumMetadata {
 
     return type
   }
+
+  @_spi(Reflection) public func caseName(forTag tag: UInt32) -> String? {
```

Comment:
I added this originally to use, but decided it wasn't needed...still useful SPI, and maybe we'll tweak the messaging to use it directly instead...